external/pom.xml

```diff
@@ -30,6 +30,12 @@
       <version>${insights-notification-schemas-java.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
+      <version>${version.org.apache.kafka.kafka-clients}</version>
+    </dependency>
```

Comment:
@pilhuhn Do you know if is fine to bump this version? Quarkus (or some dependency of it) seems to use `2.5.0` and we require `2.7.0` if we want to allow PEM files.

cc: @gwenneg

Reply:
I just ran into a Quarkus PR about bumping Kafka to 2.7.0, see https://github.com/quarkusio/quarkus/pull/14732. Apparently they had some issues.

Reply:
Kafka got updated to 2.7.0 in Quarkus. Considering the PR content, I think it's safe to update our version to 2.7.0 while waiting for a Quarkus release, as long as we don't need to run policies-engine in native mode (Quarkus registered three classes for reflection with the update). There's also a regression on Windows (https://issues.apache.org/jira/browse/KAFKA-12190) which should not be a problem.